README.md

```diff
@@ -17,26 +19,65 @@ load("@io_buildbuddy_buildbuddy_toolchain//:deps.bzl", "buildbuddy_deps")
 
 buildbuddy_deps()
 
-load("@io_buildbuddy_buildbuddy_toolchain//:rules.bzl", "buildbuddy")
+load("@io_buildbuddy_buildbuddy_toolchain//:rules.bzl", "buildbuddy_rbe_ubuntu20_04")
 
-buildbuddy(name = "buildbuddy_toolchain")
+buildbuddy_rbe_ubuntu20_04(name = "buildbuddy_rbe_ubuntu20_04")
```

Comment:
I don't have a strong opinion on this -- but would it be better to have this always be called "buildbuddy()" and select versions with a flag parameter?

Similar to like:

```
go_download_sdk(
    name = "go_sdk_darwin",
    goarch = "amd64",
    goos = "darwin",
    version = "1.19.3",
)
```

Reply:
That seems reasonable to me. How about this sort of API?

```
buildbuddy(
    name = "buildbuddy_toolchain",
    image_repository = "gcr.io/flame-public", # default value; doesn't need to be specified explicitly
    image = "rbe-ubuntu20-04:latest", # defaults to "", which means use the executor default. Currently Ubuntu 16.04
)
```

+ @siggisim

Reply:
@siggisim after our offline chat I went with a single `container_image` prop, which we can split into `(image, tag, digest)` to determine the image tool versions (based on just the "image" part).